ds3_java_cli/src/main/java/com/spectralogic/ds3cli/command/GetFilteredObjects.java

```diff
@@ -0,0 +1,131 @@
+/*
+ * *****************************************************************************
+ *   Copyright 2014-2016 Spectra Logic Corporation. All Rights Reserved.
+ *   Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+ *   this file except in compliance with the License. A copy of the License is located at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file.
+ *   This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ *   CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ *   specific language governing permissions and limitations under the License.
+ * ***************************************************************************
+ */
+
+package com.spectralogic.ds3cli.command;
+
+import com.google.common.base.Predicate;
+import com.google.common.collect.FluentIterable;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import com.spectralogic.ds3cli.Arguments;
+import com.spectralogic.ds3cli.View;
+import com.spectralogic.ds3cli.ViewType;
+import com.spectralogic.ds3cli.exceptions.BadArgumentException;
+import com.spectralogic.ds3cli.exceptions.CommandException;
+import com.spectralogic.ds3cli.models.GetDetailedObjectsResult;
+import com.spectralogic.ds3cli.util.Utils;
+import com.spectralogic.ds3cli.views.cli.DetailedObjectsView;
+import com.spectralogic.ds3client.helpers.pagination.GetObjectsFullDetailsLoaderFactory;
+import com.spectralogic.ds3client.models.DetailedS3Object;
+import com.spectralogic.ds3client.utils.Guard;
+import com.spectralogic.ds3client.utils.collections.LazyIterable;
+
+import java.util.Date;
+import javax.annotation.Nullable;
+
+public class GetFilteredObjects extends CliCommand<GetDetailedObjectsResult> {
+
+    private ImmutableMap<String, String> filterParams;
+    private String bucketName;
+    private String prefix;
+
+    @Override
+    public CliCommand init(final Arguments args) throws Exception {
+        this.filterParams = args.getFilterParams();
+        this.bucketName = args.getBucket();
+
+        if (!Guard.isStringNullOrEmpty(args.getPrefix())) {
+            throw new BadArgumentException("'-p' prefix is not supported.");
```

Comment:
This seems like it should just be a warning instead of a exceptional condition.  Also if it is not supported why even have a prefix method variable in the first place?